# origin: Debian (herbert), #173045
# cset: n/a
# inclusion: not submitted
# description: add MTD_CONCAT dep for MTD_SC520CDP
# revision date: 2004-09-03

diff -urN kernel-source-2.4.26/drivers/mtd/maps/Config.in kernel-source-2.4.26-1/drivers/mtd/maps/Config.in
--- kernel-source-2.4.26/drivers/mtd/maps/Config.in	2003-06-14 00:51:34.000000000 +1000
+++ kernel-source-2.4.26-1/drivers/mtd/maps/Config.in	2003-06-01 20:43:47.000000000 +1000
@@ -19,7 +19,7 @@
 
 if [ "$CONFIG_X86" = "y" ]; then
    dep_tristate '  CFI Flash device mapped on Photron PNC-2000' CONFIG_MTD_PNC2000 $CONFIG_MTD_CFI $CONFIG_MTD_PARTITIONS
-   dep_tristate '  CFI Flash device mapped on AMD SC520 CDP' CONFIG_MTD_SC520CDP $CONFIG_MTD_CFI
+   dep_tristate '  CFI Flash device mapped on AMD SC520 CDP' CONFIG_MTD_SC520CDP $CONFIG_MTD_CFI $CONFIG_MTD_CONCAT
    dep_tristate '  CFI Flash device mapped on AMD NetSc520'  CONFIG_MTD_NETSC520 $CONFIG_MTD_CFI $CONFIG_MTD_PARTITIONS
    dep_tristate '  CFI Flash device mapped on Arcom SBC-GXx boards' CONFIG_MTD_SBC_GXX $CONFIG_MTD_CFI_INTELEXT $CONFIG_MTD_PARTITIONS
    dep_tristate '  CFI Flash device mapped on Arcom ELAN-104NC' CONFIG_MTD_ELAN_104NC $CONFIG_MTD_CFI_INTELEXT $CONFIG_MTD_PARTITIONS
